packages/core/lib/config.js

```diff
@@ -25,13 +25,20 @@ exports.getConfig = ({ untoolNamespace = 'untool', ...overrides } = {}) => {
     name: pkgData.name || basename(rootDir),
     version: pkgData.version || '0.0.0',
     mixins: [],
+    browserWhitelist: {
+      rootDir: true,
```

Comment:
`rootDir` is being used in the config loader to replace absolute paths with `.`: https://github.com/untool/untool/blob/master/packages/webpack/lib/utils/loader.js#L19
And I put the white-list here, because `rootDir` is "owned" / introduced (?) by the core (although webpack makes use of it, so maybe it's okay to move it to webpack then?).